Use PHPCR logo as collector icon

WouterJ opened the pull request:
Currently, the same icon is used for the DoctrineBundle and the DoctrinePhpcrBundle. This causes some confusion (especially in the toolbar). A lot of times, after the first request, all items are cached and no DBAL queries are executed anymore, however as both use the same icon, it always seems as if there still is database activity.

Screenshots
---

![screenshots](https://cloud.githubusercontent.com/assets/749025/13326139/f4d449cc-dbe4-11e5-9b2a-2b23c124e2b2.png)

dbu commented:
i like it! :+1:

dantleech commented:
+1 looks cool :)

stof commented:
Please also update the png-based icon for older versions of Symfony: https://github.com/doctrine/DoctrinePHPCRBundle/blob/6d47b039d37483a33c2e70b50b8ad9ffe06ac166/Resources/views/Collector/phpcr.html.twig#L8

stof commented on Resources/views/Collector/icon.svg:
```diff
@@ -1,4 +1,3 @@
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
-    <path fill="#AAAAAA" d="M5,8h14c1.7,0,3-1.3,3-3s-1.3-3-3-3H5C3.3,2,2,3.3,2,5S3.3,8,5,8z M18,3.6c0.8,0,1.5,0.7,1.5,1.5S18.8,6.6,18,6.6s-1.5-0.7-1.5-1.5S17.2,3.6,18,3.6z M19,9H5c-1.7,0-3,1.3-3,3s1.3,3,3,3h14c1.7,0,3-1.3,3-3S20.7,9,19,9z M18,13.6
-    c-0.8,0-1.5-0.7-1.5-1.5s0.7-1.5,1.5-1.5s1.5,0.7,1.5,1.5S18.8,13.6,18,13.6z M19,16H5c-1.7,0-3,1.3-3,3s1.3,3,3,3h14c1.7,0,3-1.3,3-3S20.7,16,19,16z M18,20.6c-0.8,0-1.5-0.7-1.5-1.5s0.7-1.5,1.5-1.5s1.5,0.7,1.5,1.5S18.8,20.6,18,20.6z"/>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="20px" height="20px" viewBox="0 0 20 20" enable-background="new 0 0 20 20" xml:space="preserve">
```
``xmlns:xlink`` is even useless, so it could be removed from the SVG.

and the ``id="Layer_1"`` must be removed (as the icon might be included several times in the page, giving it an id would make the HTML invalid)

WouterJ commented:
I've fixed the SVG file and updated the PNG icon. Also, I've added a fix to remove the ghost item in the toolbar (similar to https://github.com/doctrine/DoctrineBundle/pull/503)

dbu commented:
thanks wouter!
